src/sql/src/plan/statement/ddl.rs

```diff
@@ -3069,20 +3069,34 @@ impl TryFrom<PostgresConnectionOptionExtracted>
             }
             Some(m) => sql_bail!("invalid CONNECTION: unknown SSL MODE {}", m.quoted()),
         };
+
+        // Validate that the SSH tunnel ID is indeed an SSH connection
+        let ssh_tunnel_id = self.ssh_tunnel.map(|ssh_tunnel| ssh_tunnel.into());
+        let ssh_tunnel = if let Some(ref ssh_tunnel) = ssh_tunnel_id {
```

Comment:
this is kindof hard to read, id just make the `GlobalId` in the `Object` `pub` or `pub(crate)` and just do this:
```suggestion
        let ssh_tunnel = if let Some(ssh_tunnel) = ssh_tunnel_id.map(|s| s.0) {
```

Reply:
Not sure if I got it! Either way I'd have to `map`, it just changes on whether form `x.into()` to `x.0`, no? I also need to keep the ID around because it also goes into the `PostgresConnection` further down.

Reply:
`GlobalId` is `Copy` so you can just use `.0` to create an owned version of it anywhere! Basically what I'm saying is that I prefer the `.0` to the `.into()` here! Your call, ultimately!